lib/charms/layer/basic.py

```diff
@@ -432,19 +437,19 @@ def apt_install(packages):
            '--option=Dpkg::Options::=--force-confold',
            '--assume-yes',
            'install']
-    for attempt in range(3):
+    for attempt in range(CMD_RETRY_COUNT):
         try:
             check_call(cmd + packages, env=env)
-        except CalledProcessError:
-            if attempt == 2:  # third attempt
-                raise
+        except CalledProcessError as ex:
```

Comment:
There's no good reason to use `as ex` here. i don't see where `ex` is used anywhere in the `except` clause.  Feel free to drop it and just re-raise using 

```python
raise
```